components/speedreader/speedreader_service.h

```diff
@@ -107,10 +107,20 @@ class SpeedreaderService : public KeyedService {
   SpeedreaderService& operator=(const SpeedreaderService&) = delete;
 
  private:
+  struct Setting {
+    bool is_default = false;
```

Comment:
I don't understand why this is needed, speedreader is just allow or block, right? So this is just a normal content setting that defaults to BLOCK?